ui/app/AppLayouts/Chat/CommunityColumn.qml

```diff
@@ -95,7 +95,7 @@ Item {
     ScrollView {
         id: chatGroupsContainer
         anchors.top: membershipRequests.bottom
-        anchors.topMargin: Style.current.padding
+        anchors.topMargin: Style.current.padding - 8
```

Comment:
I've added this because, with the new drag and drop capabilities, users can drag **out of** a category, To make that work, there's an additional 8px space at the top of the `StatusChatListAndCategories` component, so I'm substracting it here.

We actually need to check whether the "uncategorized" chat list is indeed empty and only then substract the height here (because only then the new drop area is active and visible)